docs/applications/gui.md

```diff
@@ -0,0 +1,53 @@
+---
+id: gui
+title: GUI
+---
+
+
+## Bitcoin-S Graphical User Interface
+There is now a GUI built using [scalafx](https://www.scalafx.org/) to allow users who do not wish to use the [command line interface](cli.md) to interact with a Bitcoin-S wallet. This GUI is currently very minimal and looks something like this:
+
+![](gui-snapshot.png)
+
+At the time of writing this document, creating addresses and sending Bitcoin are the only supported wallet functionalities in the wallet's RPC API. The expansion of this interface will enable a more robust GUI and this issue is being tracked [here](https://github.com/bitcoin-s/bitcoin-s/issues/1284).
+
+### Running the GUI
+
```

Comment:
It seems this only works with java versions 11+, maybe make a note of that

Reply:
Also should make a note that you need `bitcoin-s-server` running

Reply:
Def need the note about `bitcoin-s-server` but I'm running with `java 8` and it works just fine? Did you run into an issue?

Reply:
Yeah I ran it with `java 8` and it said that the scalafx was compiled with a version of java that was 55+ and I was using a version that only supported a version that is 52+. After I installed java 11 it was fixed

Reply:
can you try with 8 again and do a clean and run again? I have seen that error before but only when trying to package the gui using graalvm. I have no issues running it personally

Reply:
Looks like `sbt clean` fixed it :)